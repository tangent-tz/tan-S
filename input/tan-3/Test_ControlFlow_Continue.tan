main {
    {
        var i := 0; 
        while(i < 5) {
            if (i == 3) {
                i := i + 1;
                continue; 
            }
            print i \t;
            i := i + 1;
        }
        print "end of test 1." \n\n; 
    }
    

    
    {
        var outerVar := 10; 
        while (outerVar <= 70) {
            print outerVar \s ":" \t;
            
            var i := 0; 
            while(i < 5) {
                if (i == 3) {
                    i := i + 1;
                    continue; 
                }
                print i \t;
                i := i + 1;
            }
            print "end inner loop" \n;
            
            outerVar := outerVar + 10; 
        }
   
        print "end of test 2." \n;
    }
}