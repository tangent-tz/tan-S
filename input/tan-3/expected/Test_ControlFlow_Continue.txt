0	1	2	4	end of test 1.

10 :	0	1	2	4	end inner loop
20 :	0	1	2	4	end inner loop
30 :	0	1	2	4	end inner loop
40 :	0	1	2	4	end inner loop
50 :	0	1	2	4	end inner loop
60 :	0	1	2	4	end inner loop
70 :	0	1	2	4	end inner loop
end of test 2.
